{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2025-27778",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "state": "PUBLISHED",
    "assignerShortName": "GitHub_M",
    "dateReserved": "2025-03-06T18:06:54.460Z",
    "datePublished": "2025-03-19T20:42:19.866Z",
    "dateUpdated": "2025-03-19T20:42:19.866Z"
  },
  "containers": {
    "cna": {
      "title": "Applio allows unsafe deserialization in infer.py",
      "problemTypes": [
        {
          "descriptions": [
            {
              "cweId": "CWE-502",
              "lang": "en",
              "description": "CWE-502: Deserialization of Untrusted Data",
              "type": "CWE"
            }
          ]
        }
      ],
      "metrics": [
        {
          "cvssV4_0": {
            "attackVector": "NETWORK",
            "attackComplexity": "LOW",
            "attackRequirements": "NONE",
            "privilegesRequired": "NONE",
            "userInteraction": "NONE",
            "vulnConfidentialityImpact": "HIGH",
            "vulnIntegrityImpact": "HIGH",
            "vulnAvailabilityImpact": "HIGH",
            "subConfidentialityImpact": "NONE",
            "subIntegrityImpact": "NONE",
            "subAvailabilityImpact": "NONE",
            "baseScore": 8.9,
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:H/VA:H/SC:N/SI:N/SA:N/E:P",
            "version": "4.0"
          }
        }
      ],
      "references": [
        {
          "name": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/",
          "tags": [
            "x_refsource_CONFIRM"
          ],
          "url": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/"
        },
        {
          "name": "https://github.com/IAHispano/Applio/commit/16019befdcbbff0b264a5e30785feef4b70df8d9",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/commit/16019befdcbbff0b264a5e30785feef4b70df8d9"
        },
        {
          "name": "https://github.com/IAHispano/Applio/commit/eb21d9dd349a6ae1a28c440b30d306eafba65097",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/commit/eb21d9dd349a6ae1a28c440b30d306eafba65097"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/rvc/infer/infer.py#L464",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/rvc/infer/infer.py#L464"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/inference/inference.py#L338-L345",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/inference/inference.py#L338-L345"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/tts/tts.py#L50-L57",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/tts/tts.py#L50-L57"
        }
      ],
      "affected": [
        {
          "vendor": "IAHispano",
          "product": "Applio",
          "versions": [
            {
              "version": "<= 3.2.8-bugfix",
              "status": "affected"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2025-03-19T20:42:19.866Z"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "Applio is a voice conversion tool. Versions 3.2.8-bugfix and prior are vulnerable to unsafe deserialization in `infer.py`. The issue can lead to remote code execution. As of time of publication, a fix is available on the `main` branch of the Applio repository but not attached to a numbered release."
        }
      ],
      "source": {
        "advisory": "GHSA-2j77-5637-2jg7",
        "discovery": "UNKNOWN"
      }
    }
  }
}